app/src/main/java/com/github/Jenjamin3000/bootcamp/RulesFragment.java

```diff
@@ -0,0 +1,95 @@
+package com.github.Jenjamin3000.bootcamp;
+
+import static android.content.ContentValues.TAG;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link RulesFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+
+// TODO refactor to inject the DB
+public class RulesFragment extends Fragment {
+
+    FirebaseDatabase database = FirebaseDatabase.getInstance();
```

Comment:
How does this work? Is this one single static instance over the whole application which can be retrieved every time?